DFC.FindACourseClient/Models/APIRequests/CourseSearchRequest.cs

```diff
@@ -0,0 +1,36 @@
+﻿using DFC.FindACourseClient.Models.APIResponses.CourseGet.Enums;
+using System.Collections.Generic;
+
+namespace DFC.FindACourseClient.Models.APIRequests
+{
+    public class CourseSearchRequest
+    {
+        public string SubjectKeyword { get; set; }
+
+        public string DfE1619Funded { get; set; }
```

Comment:
think this is a deprecated can be removed from model

Reply:
Removed as suggested